hrui/tests/phpunit/HRUITrait.php

```diff
@@ -8,17 +8,19 @@
   /**
    * Creates a single (Individuals) contact from the provided data.
    *
-   * @param array $params should contain first_name and last_name
-   * @return int return the contact ID
-   * @throws \CiviCRM_API3_Exception
+   * @param $firstName
+   * @param $lastName
+   *
+   * @return mixed
    */
-  protected function createContact($params) {
+  protected function createContact($firstName, $lastName) {
     $result = civicrm_api3('Contact', 'create', array(
```

Comment:
A problem with this function:

- You can only create a Contact with first_name and last_name, In case there's a need to pass in other parameters when creating a contact, It can't be passed.

I think the function is supposed to work like a fabricator with some default parameters that can be overridden and the parameters passed straight to the contact API.
I think you should use the contact fabricator [Here](https://github.com/civicrm/civihr/blob/8602dcdb97bd5aa4471db45af68b70d4aac58262/uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/Contact.php#L12-L12), you will need to install the hrcore extension to be able to use it.
 Another alternative is to refactor the method to support passing in any parameter to create a contact.
 I will go with using the fabricator.